src/bundle/Resources/config/services.yaml

```diff
@@ -109,8 +106,45 @@ services:
             - { name: "support_tools.system_info.collector", identifier: "symfony_kernel" }
 
     # SystemInfoOutputFormats
-
     support_tools.system_info.output_format.json:
         class: "%support_tools.system_info.output_format.json.class%"
         tags:
             - { name: "support_tools.system_info.output_format", format: "json" }
+
+    # Gateways
+    EzSystems\EzSupportTools\Storage\MetricsProvider:
+        alias: EzSystems\EzSupportTools\Storage\AggregateMetricsProvider
+
+    EzSystems\EzSupportTools\Storage\AggregateMetricsProvider:
+        arguments:
+            $metrics: !tagged_iterator { tag: ibexa.system_info.metrics, index_by: identifier }
+
+    EzSystems\EzSupportTools\Storage\Metrics\RepositoryConnectionAwareMetrics:
+        abstract: true
+        arguments:
+            $connection: '@ezpublish.persistence.connection'
+
+    EzSystems\EzSupportTools\Storage\Metrics\PublishedContentObjectsCountMetrics:
+        autowire: true
+        tags:
+            - { name: ibexa.system_info.metrics, identifier: published }
```

Comment:
We can also consider using `!php/const`, so the identifier usage can be more easily found throughout codebase if we ever change them.

https://symfony.com/blog/new-in-symfony-3-2-php-constants-in-yaml-files

Reply:
Fixed via e9785ac0 though not sold on readability of this solution. I hope further IDE upgrades would provide at least refactoring for that.